### DISCUSSION TITLE
Who uses React Cosmos?

### DISCUSSION THREAD

**Discussion by ovidiuch** [react-cosmos/react-cosmos#1612] [Show and tell]
We would like to gather a list of projects/companies using React Cosmos, for the website.

Let us know if your project/company uses React Cosmos and wouldn't mind being mentioned on the website.

Include a logo with a link to the project if you have one. **Please try your best to provide an SVG logo** – SVGs look great on any screen size and are easier to integrate on the React Cosmos homepage compared to other image format.

A short description on how you use React Cosmos and how it has helped you is also welcome.

Thank you!

**Comment by MirzayevFarid** [comment]
- We’re using React Cosmos at [Radity](https://radity.com/)

- ![logo-midnight@3x](https://user-images.githubusercontent.com/29518613/91811146-173a9a00-ec40-11ea-9d20-981cfcf59877.png)

- https://radity.com
- Radity develops digital products and build web and mobile applications.

Thanks for such an amazing work!

**Comment by tolunayakbulut** [comment]
We prefer React Cosmos at Global CTO Forum.

Here is our logo and website.

https://globalctoforum.org/

![global-cto-forum](https://user-images.githubusercontent.com/31761883/101292916-7ef61780-3823-11eb-9a8e-ac924ff7abf4.png)

Thank you for developing

**Comment by tsanyqudsi** [comment]
currently using React Cosmos on VivelJS.

https://github.com/viveljs/viveljs

**Comment by onino-online** [comment]
I took my time to pick the best ui presentation library for my clients and I came up with react cosmos a few years ago.

I use react cosmos every day because it's not a ui presentation library, it helps me to create components (ui or not) in isolation independently from my project. It brings a lot of quality to my dev and eventually we can use it as a presentation library if one is needed. I imposed it to my team members and everybody agree ith this decision.

I'm working on business application for large companies like:

![1200px-Geodis svg (Téléphone)](https://github.com/react-cosmos/react-cosmos/assets/117453947/84b62363-5baa-47e1-84b3-fd3405544bb7)
![TECHNIP_ENERGIES_LOGO (Téléphone)](https://github.com/react-cosmos/react-cosmos/assets/117453947/2fff8451-0b44-4f5b-8419-e09c1dec9f6f)

I would be very annoyed if the librairy was no longer maintained.

Thanks a lot for this huge work !

**Comment by ovidiuch** [comment]
@CodeJPB That's great to hear!

Happy to add those logos to the website, but could you provide SVG options for them? SVGs look much better regardless of the screen resolution and are easier to style using a hue filter to blend well with the website. Thanks!
